src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -52,13 +52,22 @@ public void setPersons(List<Person> persons) {
         this.persons.setPersons(persons);
     }
 
+    /**
+     * Replaces the contents of the person list with {@code persons}.
+     * {@code persons} must not contain duplicate persons.
+     */
+    public void setMeetings(List<Meeting> meetings) {
+        this.meetings.setInternalList(meetings);
+    }
+
     /**
      * Resets the existing data of this {@code AddressBook} with {@code newData}.
      */
     public void resetData(ReadOnlyAddressBook newData) {
         requireNonNull(newData);
 
         setPersons(newData.getPersonList());
+        setMeetings(newData.getMeetingList());
     }
 
     //// person-level operations
```

Comment:
Why does this have comment in a comment?

Reply:
I did not add it ok

Reply:
Urm is this urm fixed?